app/uk/gov/hmrc/ngrloginregisterfrontend/views/FeatureNotImplementedView.scala.html

```diff
@@ -0,0 +1,33 @@
+@*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+// $COVERAGE-OFF$
+@import uk.gov.hmrc.ngrloginregisterfrontend.views.html.components._
+@import uk.gov.hmrc.ngrloginregisterfrontend.config.AppConfig
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+
+@this(layout: Layout)
+
+@(journeyId: Option[String])(implicit request: RequestHeader, messages: Messages,  appConfig: AppConfig)
+
+@layout(pageTitle = Some(messages("featureNotAvailable.title"))) {
+
+<h1 class="govuk-heading-xl">@messages("featureNotAvailable.title")</h1>
+
```

Comment:
@if(journeyId.isDefined) {
     <p class="govuk-body">@messages("featureNotAvailable.idTrue", journeyId.getOrElse(""))</p>
} else {
     <p class="govuk-body">@messages("featureNotAvailable.idFalse")</p>
} add this to switch between the messages